src/actors.h

```diff
@@ -129,11 +129,8 @@ extern struct Actor gActorList[];
 extern Player gPlayers[];
 extern u16 gNumActors;
 extern u16 gNumPermanentActors;
-/** 
- * Incremented by one every time the player spawns a new shell actor.
- * Decremented onces 21 have been spawned
- */
-extern u16 gNumPlayerSpawnedShells; 
```

Comment:
The first line was fine. My issue was with "Decremented onces 21 have been spawned" as I thought it made more sense to say that the shells get cleanedup once 21 have spawned. Perhaps add that first line back in please.